R/readme.R

```diff
@@ -11,7 +11,13 @@
 #' added to \code{.Rbuildignore}. The resulting README is populated with default
 #' YAML frontmatter and R fenced code blocks (\code{md}) or chunks (\code{Rmd}).
 #'
+#' \code{use_data_list_in_readme_rmd} and \code{use_data_list_in_readme_md}
+#' append a table to the README listing the datasets contained in the package.
 #' @inheritParams use_template
+#' @param stop_if_no_data Logical. If \code{TRUE},
```

Comment:
Having slept on it I've reworked this so that it is no longer a user facing argument.

If the user calls `use_readme_rmd()` or `use_readme_md()`, then it fails silently.

If the user calls `use_data_list_in_readme_rmd()` or `use_data_list_in_readme_md()`, that is, to explicitly add the dataset details afterwards, then it warns if there are no datasets.